labs/lab-2-javascript-game/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script type="text/javascript" src="script.js">
+  </script>
```

Comment:
I'd keep the open and close on one line, to make it clear it's linking to a file, not being inline js.